src/cf-forms/src/cf-forms.less

```diff
@@ -219,6 +219,340 @@ textarea {
+    &[type="radio"]:disabled + label::before {
+      background: @input-bg-disabled;
+      box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-border-disabled;
+      cursor: not-allowed;
+    }
+
+}
+
+//
+// Form layouts
 //
+
+.form-l-inset {
+    border-right-width: 2.5px;
+
+    + .form-l_col-1-2.form-l-inset:nth-child(odd) { 
+        border-left-width: 2.5px;
+    }
+    
+    &.form-l_col-1-2:nth-child(odd),
+    &.form-l_col-1-2:last-child {
+        border-right-width: 15px;
+    }
+
+    .form-l-inset_container {
+        background: @input-inset-bg;
+
+        .cf-input_label {
+            margin: 0;
+            padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+               unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+               unit( ( @grid_gutter-width / 1.75 ) / @base-font-size-px, em );
+        }
+
+        .cf-input_text {
+            margin-left: unit( @grid_gutter-width / @base-font-size-px, em );
+        }
+
+        .cf-input_radio,
+        .cf-input_checkbox {
+            position: absolute;
+            top: unit( ( @grid_gutter-width / 1.75 ) / @base-font-size-px, em );
+            left: 15px;
+            z-index: 3;
+        }
+
+        &:hover,
+        &:focus,
+        &.focus {
+            .cf-input_checkbox,
+            .cf-input_radio {
+                box-shadow: 0px 0px 0px 2px @input-border-focus;
+                
+                // Older IE does not support border-radius or box-shadow it's ok to
+                // use outline here.
+                .lt-ie9 & {
+                    outline: 1px solid @input-border-focus;
+                }
+            }
+        }
+
+        &.selected .cf-input_label,
+        &.focus .cf-input_label,
+        &:focus .cf-input_label,
+        &:hover .cf-input_label,
+        input:focus + .cf-input_label,
+        input:checked + .cf-input_label     {
+            outline: 2px solid @input-border-focus;
+        }
+
+        &.selected .cf-input_label,
+        input:checked + .cf-input_label {
+            background-color: @input-inset-selected;
+        }
+    }
+}
+
+//
+// m-select menu
+//
+
+.m-select {
+    position: relative;
+    margin-top: 0.5em;
+    border: 1px solid @gray-40;
+
+    select {
+        height: unit( 30px / @base-font-size-px, em );
+        width: 100%;
+        padding: unit( 4px / @base-font-size-px, em )
+                 unit( 6px / @base-font-size-px, em );
+        border: 0;
+        outline: 2px solid transparent;
+        appearance: none;
+        background-color: @white;
+        color: @black;
+
+        &:hover,
+        &:active,
+        &:focus {
+            outline-color: @pacific;
+            outline-offset: 0;
+        }
+    }
+
+    select[disabled] {
+        color: @gray-80;
+        background-color: @gray-10;
+    }
+
+    select option:disabled {
+        color: @gray-80;
+    }
+
+    &:after {
+        position: absolute;
```

Comment:
I think I might have originally wrote this, but positioning variables go after box-model variables. Those should get rearranged.